classes/survey.class.php

```diff
@@ -44,60 +44,37 @@ public static function get_instance() {
 		 * @return array
 		 * @see survey.js
 		 */
-		public function get_survey_metadata() {
-			$license_data = get_option( 'ppom_pro_license_data', array() );
-			$attributes   = array();
-			$user_id      = 'ppom_' . ( ! empty( $license_data->key ) ? $license_data->key : preg_replace( '/[^\w\d]*/', '', get_site_url() ) ); // Use a normalized version of the site URL as a user ID for free users.
-
-			$days_since_install = round( ( time() - get_option( 'woocommerce_product_addon_install', 0 ) ) / DAY_IN_SECONDS );
-			$install_category   = 0; // Normalized value.
-			if ( 0 === $days_since_install || 1 === $days_since_install ) {
-				$install_category = 0;
-			} elseif ( 1 < $days_since_install && 8 > $days_since_install ) {
-				$install_category = 7;
-			} elseif ( 8 <= $days_since_install && 31 > $days_since_install ) {
-				$install_category = 30;
-			} elseif ( 30 < $days_since_install && 90 > $days_since_install ) {
-				$install_category = 90;
-			} elseif ( 90 <= $days_since_install ) {
-				$install_category = 91;
+		public function get_survey_metadata( $data, $page_slug ) {
+			if ( defined( 'CYPRESS_TESTING' ) ) {
+				return $data;
 			}
 
-			$attributes['days_since_install'] = strval( $install_category );
-			$attributes['license_status']     = ! empty( $license_data->license ) ? $license_data->license : 'invalid';
-			$attributes['free_version']       = PPOM_VERSION;
-
-			if ( ! empty( $license_data->plan ) ) {
-				$attributes['plan'] = $this->plan_category( $license_data );
-			}
+			$license_data = get_option( 'ppom_pro_license_data', array() );
```

Comment:
we can use $license_status               = apply_filters( 'product_ppom_license_status', '' ); for status and product_ppom_license_key for key